draft-ietf-gnap-core-protocol.md

```diff
@@ -2754,9 +2755,8 @@ sections below.
 If the AS determines that the RC can 
 make a further continuation request, the AS MUST include a new 
 ["continue" response](#response-continue). 
-If the continuation was previously bound to an access token, the
-new `continue` response MUST include a bound access token as well, and
-this token SHOULD be a new access token. 
+The new `continue` response MUST include a bound access token as well, and
+this token SHOULD be a new access token, invalidating the previous access token.
```

Comment:
Why SHOULD? Are there cases where there's a good reason to retain an old one? I am sure this detail is critical for any formal security proof.

Reply:
I fully agree this is a MUST. Dick had another view however.

Reply:
If we need to call WG consensus on this, fine. We cannot have SHOULDs just because we are unable to reach agreement. Quoting RFC 2119, "This word, or the adjective "RECOMMENDED", mean that there may exist valid reasons in particular circumstances to ignore a particular item, but the full implications must be understood and carefully weighed before choosing a different course." Which implies that *we* need to say when it may be appropriate to keep the old one.

Reply:
The best would be for Dick to reexplain his use case (and other people as well if it makes sense) and then we can decide as a group.

Reply:
This PR doesn't change the behavior here, so we'll track whether this should be a SHOULD/MUST in a separate issue: #147